WalletWasabi/Wallets/Wallet.cs

```diff
@@ -550,7 +565,7 @@ public void SetWaitingForInitState()
 	public static Wallet CreateAndRegisterServices(Network network, BitcoinStore bitcoinStore, KeyManager keyManager, WasabiSynchronizer synchronizer, string dataDir, ServiceConfiguration serviceConfiguration, HybridFeeProvider feeProvider, IBlockProvider blockProvider)
 	{
 		var wallet = new Wallet(dataDir, network, keyManager);
-		wallet.RegisterServices(bitcoinStore, synchronizer, serviceConfiguration, feeProvider, blockProvider);
+		wallet.RegisterServices(bitcoinStore, synchronizer, serviceConfiguration, feeProvider, blockProvider, coinPrison: new(""));
```

Comment:
`new("")` Are you sure? is this only used in tests?

Reply:
Yes, it's for testing.